app/uk/gov/hmrc/eoricommoncomponent/frontend/services/cache/sessionCache.scala

```diff
@@ -16,20 +16,18 @@
 
```

Comment:
This class radically alters the API into the persistence layer. This is not ideal as it creates a maintenance overhead. In hindsight it would have been preferable to insert an abstraction layer to decouple the service and persistence layers. Please a) add a tech debt ticket to address this issue. and b) add comments to explain why this extensive change was necessary and why it was done this way. This will aid future work in this area of the codebase.

Reply:
I have added the comments in the ticket for using SessionCacheRepository